src/module-magespices-mage2dark/etc/adminhtml/di.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Theme\Model\View\Design">
+        <plugin sortOrder="1" name="mageSpicesMage2DarkDesign" type="MageSpices\Mage2Dark\Plugin\Model\View\Design"/>
```

Comment:
`sortOrder` for plugins are discouraged (whatever the order is - the results should be the same)

Reply:
will remove it, it was added by magicento :-)

Reply:
Done ✅